src/Trakx.Kaiko.ApiClient.Tests/Integration/EnabledServices.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Trakx.Kaiko.ApiClient.Tests;
+
+internal static class EnabledServices
+{
+    internal const bool AggregateVwap = false;
```

Comment:
Should these settings be static in the code? If this must be changed, it will be needed to release a new version of this lib.